### PR TITLE
Update chatbees_script.js to handle errors better

### PR DESCRIPTION
**Pull request by jon-snowman**

### CODE REVIEW

File: chatbees_script.js [hunk 1]
```diff
@@ -113,9 +118,12 @@ function chatbeesSendMessage() {
...
   .catch(error => {
       console.error('Error:', error);
-      // Display the error in the chat area
+      // remove the thinking message
+      chatArea.removeChild(thinkMsg);
+
+      // Display a generic message for error case
       var botMsg = document.createElement('div');
-      botMsg.textContent = error;
+      botMsg.textContent = "Something went wrong, please try again later";
```
Comment (JuniusLuo) [inline chatbees_script.js]: why not show the error? if customer reports issue, we will have to ask them to check console log, which may be hard for non-tech people, also not easy on mobile.

File: chatbees_script.js [hunk 2]
```diff
@@ -94,10 +94,15 @@ function chatbeesSendMessage() {
     },
     body: jsonData,
   })
-  .then(response => response.json())
+  .then((response) => {
+      if (response.ok) {
+         return response.json();
+      }
+      return Promise.reject(response);
```
Comment (JuniusLuo) [inline chatbees_script.js]: can we return the error in the response?

File: chatbees_script.js (hunk 1 above)
Comment (jon-snowman) [reply chatbees_script.js]: sg, appended the error message

File: chatbees_script.js (hunk 2 above)
Comment (jon-snowman) [reply chatbees_script.js]: yup, added.

**Review by JuniusLuo — APPROVED** [review]

**Pull request merged**